### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by sbansiwal**
first comment

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,8 +2,4 @@ File to test webhook events
 changed with new branch (test-branch)
 modifications
 test file
-check file
-make changes
-commit 2
-draft check 
-line check
+check comments
```
Comment (sbansiwal) [inline README.md:5]: second comment

File: README.md [hunk 2]
```diff
@@ -2,8 +2,4 @@ File to test webhook events
 changed with new branch (test-branch)
 modifications
```
Comment (sbansiwal) [inline README.md:3]: third comment

**Review by sbansiwal — COMMENTED** [review]
fourth comment

File: README.md [hunk 3]
```diff
@@ -2,8 +2,4 @@ File to test webhook events
 changed with new branch (test-branch)
```
Comment (Revant832) [inline README.md:2]: fifith comment

Comment (Revant832) [inline README.md:2]: sixth comment

**Review by Revant832 — COMMENTED** [review]
seventh comment

**Pull request merged**